src/api/app/views/webui/user/_info.html.haml

```diff
@@ -33,6 +33,9 @@
             %i.fa.fa-question-circle.text-info{ data: { placement: 'top', toggle: 'popover', html: 'true',
                                                         content: 'Join the <strong>beta program</strong> to try the latest ' + |
                                                                  'features we develop and give us feedback on them before they go live.' } } |
+      - if user.in_beta?
+        .mt-2
+          = link_to('Opt out of beta features', my_opted_out_features_path)
```

Comment:
Please don't use technical terms as 'opt out' in user facing language.

Reply:
What do you think of `Disable beta features`?

Reply:
I've changed this to `Disable beta features`.

Reply:
@dmarcoux, WDYT about `Manage beta features` or something similar, because you can enable and disable them.

Reply:
Thanks for the suggestion, this is better :)